media-libs/raylib/raylib-5.0.recipe

```diff
@@ -0,0 +1,76 @@
+SUMMARY="A simple and easy-to-use library to enjoy videogames programming"
+DESCRIPTION="Raylib is highly inspired by Borland BGI graphics \
+lib and by XNA framework and it's especially well suited for \
+prototyping, tooling, graphical applications, embedded systems and education."
+HOMEPAGE="https://github.com/raysan5/raylib"
+COPYRIGHT="2013-2024 Ramon Santamaria"
+LICENSE="Zlib"
+REVISION="1"
+SOURCE_URI="https://github.com/raysan5/raylib/archive/refs/tags/$portVersion.tar.gz"
+CHECKSUM_SHA256="98f049b9ea2a9c40a14e4e543eeea1a7ec3090ebdcd329c4ca2cf98bc9793482"
+SOURCE_FILENAME="$portVersion.tar.gz"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+libVersion="4.5.0"
+libVersionCompat="$libVersion compat >= ${libVersion%.**}"
+
+PROVIDES="
+	raylib$secondaryArchSuffix = $portVersion
+	lib:libraylib$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libGL$secondaryArchSuffix
+	lib:libGLU$secondaryArchSuffix
+	lib:libglfw$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	raylib${secondaryArchSuffix}_devel = $portVersion
+	devel:libraylib${secondaryArchSuffix} = $libVersionCompat
+	"
+REQUIRES_devel="
+	raylib$secondaryArchSuffix == $portVersion base
+	devel:libGL$secondaryArchSuffix
+	devel:libGLU$secondaryArchSuffix
+	devel:libglfw$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libGL$secondaryArchSuffix
+	devel:libGLU$secondaryArchSuffix
+	devel:libglfw${secondaryArchSuffix}
+	"
+BUILD_PREREQUIRES="
+	cmd:gcc$secondaryArchSuffix
+	cmd:cmake
+	cmd:make
+	"
+
+BUILD()
+{
+	mkdir -p build
+	cd build
+
+	cmake .. \
+		-DCMAKE_BUILD_TYPE=Release \
+		$cmakeDirArgs \
+		-DUSE_EXTERNAL_GLFW=ON \
+		-DBUILD_SHARED_LIBS=ON
+
+	make $jobArgs
+}
+
+INSTALL()
+{
+	cd build
+	make install
+
+	prepareInstalledDevelLibs \
+		libraylib
+
```

Comment:
Looks good!

Could you add

```sh
fixPkgconfig
```

so that it is moved to the devel package?

Reply:
Hope we don't scare you :) Another small thing, only one library so:
```diff
-	prepareInstalledDevelLibs \
+	prepareInstalledDevelLib \
```

Reply:
updated to include `fixPkgConfig` and `lib`

i don't understand cmake conventions very well (if at all), so the feedback is helpful!

Reply:
For the above comments cmake is not involved, in case you are wondering about how $cmakeDirArgs work, this takes care of all the default installation paths, so you don't need to declare them all in a recipe, eg not have to worry about -DCMAKE_INSTALL_PREFIX/-DCMAKE_INSTALL_BINDIR etc.